antsibull/data/collection-enum.py

```diff
@@ -7,17 +7,19 @@
 
 import argparse
 import json
+import os
 import sys
 
+import yaml
+
 import ansible.plugins.loader as plugin_loader
 
 from ansible import constants as C
 from ansible import release as ansible_release
 from ansible.cli import doc
 from ansible.cli.arguments import option_helpers as opt_help
 from ansible.collections.list import list_collection_dirs
-from ansible.galaxy.collection import CollectionRequirement
-from ansible.module_utils._text import to_native
+from ansible.module_utils._text import to_native, to_bytes
```

Comment:
`to_bytes` is unnecessary per comments below
```suggestion
from ansible.module_utils._text import to_native
```